translation: SyntheticEvent

ps0305 opened the pull request:
<!--

Thank you for the PR! Contributors like you keep React awesome!

Please see the Contribution Guide for guidelines:

https://github.com/reactjs/reactjs.org/blob/master/CONTRIBUTING.md

If your PR references an existing issue, please add the issue number below

-->
 Hindi Translation Progress #1 

Translation for API Reference => SyntheticEvent

bvaughn commented:
Sorry, I can't review this. I don't speak Hindi. :smile:

saranshkataria commented:
@bvaughn never too late to learn? ;)
We will do it in the meantime.

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -1,18 +1,18 @@
 ---
 
-This reference guide documents the `SyntheticEvent` wrapper that forms part of React's Event System. See the [Handling Events](/docs/handling-events.html) guide to learn more.
+यह संदर्भ React इवेंट सिस्टम का हिस्सा बनने वाले `SyntheticEvent` आवरण को निर्देशित करता है. अधिक जानने के लिए मार्गदर्शिका देखें | [Handling Events](/docs/handling-events.html)
```
`अधिक जानने के लिए मार्गदर्शिका देखें | [Handling Events](/docs/handling-events.html)` => `अधिक जानने के लिए [Handling Events](/docs/handling-events.html) गाइड  देखें।`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -1,18 +1,18 @@
+यह संदर्भ React इवेंट सिस्टम का हिस्सा बनने वाले `SyntheticEvent` आवरण को निर्देशित करता है. अधिक जानने के लिए मार्गदर्शिका देखें | [Handling Events](/docs/handling-events.html)
 
-## Overview {#overview}
+## अवलोकन {#overview}
```
`अवलोकन` => `ओवरव्यू`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -1,18 +1,18 @@
+आपके ईवेंट हैंडलर को `SyntheticEvent` के उदाहरण दिए जाएंगे, जो ब्राउज़र के मूल ईवेंट के आसपास एक क्रॉस-ब्राउज़र आवरण है। यह ब्राउज़र के मूल ईवेंट के समान है, जिसमें `stopPropagation()` और `stopDefault()` शामिल हैं, सिवाय उन घटनाओं के, जो सभी ब्राउज़रों में समान रूप से काम करते हैं।
 
-If you find that you need the underlying browser event for some reason, simply use the `nativeEvent` attribute to get it. Every `SyntheticEvent` object has the following attributes:
+यदि आप पाते हैं कि आपको किसी कारण से अंतर्निहित ब्राउज़र इवेंट की आवश्यकता है, तो इसे प्राप्त करने के लिए केवल `nativeEvent` विशेषता का उपयोग करें। प्रत्येक `SyntheticEvent` object में निम्न विशेषताएँ होती हैं।
```
`object` => `ऑब्जेक्ट`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -1,18 +1,18 @@
+## अवलोकन {#overview}
 
-Your event handlers will be passed instances of `SyntheticEvent`, a cross-browser wrapper around the browser's native event. It has the same interface as the browser's native event, including `stopPropagation()` and `preventDefault()`, except the events work identically across all browsers.
+आपके ईवेंट हैंडलर को `SyntheticEvent` के उदाहरण दिए जाएंगे, जो ब्राउज़र के मूल ईवेंट के आसपास एक क्रॉस-ब्राउज़र आवरण है। यह ब्राउज़र के मूल ईवेंट के समान है, जिसमें `stopPropagation()` और `stopDefault()` शामिल हैं, सिवाय उन घटनाओं के, जो सभी ब्राउज़रों में समान रूप से काम करते हैं।
```
`stopDefault()` => `preventDefault()`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,13 @@ number timeStamp
+> ध्यान दें:
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> V0.14 के अनुसार, किसी इवेंट हैंडलर से `false` वापस करना अब event propagation को रोक नहीं पाएगा। इसके बजाय, `e.stopPropagation()` या `e.preventDefault()` को मैन्युअल रूप से, उपयुक्त के रूप में चालू/ट्रिगर किया जाना चाहिए।
```
`V0.14` => `v0.14`

arshadkazmi42 replied:
```
इसके बजाय, `e.stopPropagation()` या `e.preventDefault()` को मैन्युअल रूप से, उपयुक्त के रूप में चालू/ट्रिगर किया जाना चाहिए।
```
=>
```
इसके बजाय, `e.stopPropagation()` या `e.preventDefault()` को ज़रूरत के अनुसार मैन्युअली स्टार्ट किया जाना चाहिए।
```

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,13 @@ number timeStamp
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` को इकट्ठा किया जाता है। इसका मतलब यह है कि `SyntheticEvent` object का पुन: उपयोग किया जाएगा और ईवेंट कॉलबैक लागू होने के बाद सभी गुणों को null कर दिया जाएगा। यह निष्पादन कारणों से है। जैसे, आप ईवेंट को asynchronous तरीके से एक्सेस नहीं कर सकते.
```
`object` => `ऑब्जेक्ट`

arshadkazmi42 replied:
`गुणों` => `प्रॉपर्टीज`

arshadkazmi42 replied:
`null` => `nullify`

arshadkazmi42 replied:
`कर सकते.` => `कर सकते।`

arshadkazmi42 replied:
add line break before this `यह निष्पादन कारणों से है।`

and 1 line break before this `जैसे, आप ईवेंट को asynchronous तरीके से एक्सेस नहीं कर सकते`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,13 @@ number timeStamp
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` को इकट्ठा किया जाता है। इसका मतलब यह है कि `SyntheticEvent` object का पुन: उपयोग किया जाएगा और ईवेंट कॉलबैक लागू होने के बाद सभी गुणों को null कर दिया जाएगा। यह निष्पादन कारणों से है। जैसे, आप ईवेंट को asynchronous तरीके से एक्सेस नहीं कर सकते.
```
`निष्पादन` => `परफॉरमेंस`

arshadkazmi42 replied:
Can you remove line break at line 41.
I think line break was added with this fix

arshadkazmi42 replied:
> Can you remove line break at line 41.
> I think line break was added with this fix

ignore this. my bad. 
that was needed

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -60,15 +58,15 @@ function onClick(event) {
+> ध्यान दें:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> यदि आप एक asynchronous तरीके से इवेंट के गुणों का उपयोग करना चाहते हैं, तो आपको इवेंट पर `event.persist()` को कॉल करना चाहिए, जो पूल(इकट्ठा इवेंट) से `SyntheticEvent` को हटा देगा और उपयोगकर्ता कोड द्वारा इवेंट के संदर्भ को बनाए रखने की अनुमति देगा।
```
`गुणों` => `प्रॉपर्टीज`

arshadkazmi42 replied:
`पूल(इकट्ठा इवेंट)` => `पूल`

arshadkazmi42 replied:
```
`SyntheticEvent`
```
=>
```
सिंथेटिक इवेंट
```

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -60,15 +58,15 @@ function onClick(event) {
+## समर्थित इवेंट्स {#supported-events}
 
-React normalizes events so that they have consistent properties across different browsers.
+React घटनाओं को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
```
`घटनाओं` => `इवेंट्स`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -60,15 +58,15 @@ function onClick(event) {
+React घटनाओं को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
 
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+नीचे दिए गए घटना संचालकों को bubbling चरण में एक इवेंट द्वारा ट्रिगर किया गया है। कैप्चर/पकड़ना चरण के लिए एक ईवेंट हैंडलर पंजीकृत करने के लिए, `Capture` को ईवेंट नाम में जोड़ें, उदाहरण के लिए, `onClick` का उपयोग करने के बजाय, आप कैप्चर/पकड़ना चरण में click इवेंट को संभालने के लिए `onClickCapture` का उपयोग करेंगे।
```
`कैप्चर/पकड़ना` => `कैप्चर`

arshadkazmi42 replied:
`पंजीकृत` => `रजिस्टर`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -60,15 +58,15 @@ function onClick(event) {
+React घटनाओं को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
 
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+नीचे दिए गए घटना संचालकों को bubbling चरण में एक इवेंट द्वारा ट्रिगर किया गया है। कैप्चर/पकड़ना चरण के लिए एक ईवेंट हैंडलर पंजीकृत करने के लिए, `Capture` को ईवेंट नाम में जोड़ें, उदाहरण के लिए, `onClick` का उपयोग करने के बजाय, आप कैप्चर/पकड़ना चरण में click इवेंट को संभालने के लिए `onClickCapture` का उपयोग करेंगे।
```
`कैप्चर/पकड़ना` => `कैप्चर`

arshadkazmi42 replied:
you missed this

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -89,33 +87,33 @@ The event handlers below are triggered by an event in the bubbling phase. To reg
 ```
 
-Properties:
+गुण:
````
`गुण` => `प्रॉपर्टीज`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -89,33 +87,33 @@ The event handlers below are triggered by an event in the bubbling phase. To reg
 * * *
 
-### Composition Events {#composition-events}
+### संयोजन इवेंट्स {#composition-events}
```
`संयोजन` => `कम्पोजीशन`

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -89,33 +87,33 @@ The event handlers below are triggered by an event in the bubbling phase. To reg
 ```
 
-Properties:
+गुण:
````
`गुण` => `प्रॉपर्टीज`

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -124,15 +122,15 @@ string data
 ```
 
-Properties:
+गुण:
````
`गुण` => `प्रॉपर्टीज`

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -149,53 +147,53 @@ boolean shiftKey
 ```
 
-The `key` property can take any of the values documented in the [DOM Level 3 Events spec](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
+`key` संपत्ति, [DOM स्तर 3 इवेंट्स विनिर्देश](https://www.w3.org/TR/uievents-key/#named-key-attribute-values) में प्रलेखित किसी भी मान को ले सकती है।
````
`संपत्ति` => `प्रॉपर्टीज`

arshadkazmi42 replied:
`मान` => `वैल्यू`

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -149,53 +147,53 @@ boolean shiftKey
 ```
 
-These focus events work on all elements in the React DOM, not just form elements.
+ये फ़ोकस इवेंट React DOM में सभी elements पर काम करते हैं, न कि केवल फार्म elements।
````
`elements` => `एलिमेंट्स`

arshadkazmi42 replied:
`elements` => `एलिमेंट्स`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -149,53 +147,53 @@ boolean shiftKey
+ये फ़ोकस इवेंट React DOM में सभी elements पर काम करते हैं, न कि केवल फार्म elements।
 
-Properties:
+गुण:
```
`गुण` => `प्रॉपर्टीज`

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -149,53 +147,53 @@ boolean shiftKey
 ```
 
-For more information about the onChange event, see [Forms](/docs/forms.html).
+OnChange ईवेंट के बारे में अधिक जानकारी के लिए [फार्म](/docs/forms.html) देखें।
````
`OnChange` => `onChange`

arshadkazmi42 replied:
`फार्म` => `फॉर्म्स`

ps0305 commented:
> 
> 
> Great start.
> 
> I have added comments till line 193. Lets get these closed, then I will start with further reviews.
> Also refer #23 to know more about our review process

updated as marked, can go ahead and review further.

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -149,53 +149,53 @@ boolean shiftKey
 ```
 
-The `onMouseEnter` and `onMouseLeave` events propagate from the element being left to the one being entered instead of ordinary bubbling and do not have a capture phase.
+`onMouseEnter` और` onMouseLeave` ईवेंट उस एलिमेंट से प्रचारित होते हैं जिसे साधारण बब्बलिङ्ग के बजाय किसी एक में छोड़ा जा रहा है और कैप्चर फेज नहीं है।
````
```
और` onMouseLeave`
```
=>
```
और `onMouseLeave`
```

spacing issue

arshadkazmi42 replied:
`बब्बलिङ्ग` => `bubbling`

arshadkazmi42 commented on content/docs/reference-events.md:
````diff
@@ -216,20 +216,20 @@ boolean shiftKey
 ```
 
-The `onPointerEnter` and `onPointerLeave` events propagate from the element being left to the one being entered instead of ordinary bubbling and do not have a capture phase.
+`onPointerEnter` और` onPointerLeave` ईवेंट उस एलिमेंट से प्रचारित होते हैं जिसे साधारण बब्बलिङ्ग के बजाय एक में छोड़ा जा रहा है और कैप्चर फेज नहीं है।
````
```
और` onPointerLeave`
```
=>
```
और `onPointerLeave`
```

spacing issue

arshadkazmi42 replied:
`बब्बलिङ्ग` => `bubbling`

arshadkazmi42 commented on content/docs/reference-events.md:
```diff
@@ -244,33 +244,33 @@ string pointerType
+क्रॉस-ब्राउज़र समर्थन पर ध्यान दें:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+पॉइंटर ईवेंट्स अभी तक हर ब्राउज़र में समर्थित नहीं हैं (इस लेख को लिखने के समय, समर्थित ब्राउज़र में शामिल हैं: क्रोम, फ़ायरफ़ॉक्स, एज और इंटरनेट एक्सप्लोरर)। React जानबूझकर अन्य ब्राउज़रों के लिए पॉलीफिल समर्थन नहीं करता है क्योंकि एक मानक-अनुरूप पॉलीफिल `react-dom` के बंडल आकार में काफी वृद्धि करेगा।
```
`क्रोम, फ़ायरफ़ॉक्स, एज और इंटरनेट एक्सप्लोरर` => `Chrome, Firefox, Edge और Internet Explorer`

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -6,13 +6,13 @@ layout: docs
 ---
 
-This reference guide documents the `SyntheticEvent` wrapper that forms part of React's Event System. See the [Handling Events](/docs/handling-events.html) guide to learn more.
+यह संदर्भ React इवेंट सिस्टम का हिस्सा बनने वाले `SyntheticEvent` आवरण को निर्देशित करता है। अधिक जानने के लिए [Handling Events](/docs/handling-events.html) गाइड देखें।
```
यह संदर्भ React इवेंट सिस्टम का हिस्सा बनने वाले `SyntheticEvent` आवरण को निर्देशित करता है। -> could be simplified a bit? too complex

saranshkataria replied:
आवरण could be wrapper in hindi

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -6,13 +6,13 @@ layout: docs
+## ओवरव्यू {#overview}
 
-Your event handlers will be passed instances of `SyntheticEvent`, a cross-browser wrapper around the browser's native event. It has the same interface as the browser's native event, including `stopPropagation()` and `preventDefault()`, except the events work identically across all browsers.
+आपके ईवेंट हैंडलर को `SyntheticEvent` के उदाहरण दिए जाएंगे, जो ब्राउज़र के मूल ईवेंट के आसपास एक क्रॉस-ब्राउज़र आवरण है। यह ब्राउज़र के मूल ईवेंट के समान है, जिसमें `stopPropagation()` और `preventDefault()` शामिल हैं, सिवाय उन घटनाओं के, जो सभी ब्राउज़रों में समान रूप से काम करते हैं।
```
उदाहरण -> incorrect translation

saranshkataria replied:
मूल -> native in hindi

saranshkataria replied:
जो ब्राउज़र के मूल ईवेंट के आसपास एक क्रॉस-ब्राउज़र आवरण है। -> revisit

saranshkataria replied:
revisit the last line as well, it changes the meaning of the original sentence

saranshkataria replied:
wrapper -> आवरण for consistency?

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -6,13 +6,13 @@ layout: docs
+आपके ईवेंट हैंडलर को `SyntheticEvent` के उदाहरण दिए जाएंगे, जो ब्राउज़र के मूल ईवेंट के आसपास एक क्रॉस-ब्राउज़र आवरण है। यह ब्राउज़र के मूल ईवेंट के समान है, जिसमें `stopPropagation()` और `preventDefault()` शामिल हैं, सिवाय उन घटनाओं के, जो सभी ब्राउज़रों में समान रूप से काम करते हैं।
 
-If you find that you need the underlying browser event for some reason, simply use the `nativeEvent` attribute to get it. Every `SyntheticEvent` object has the following attributes:
+यदि आप पाते हैं कि आपको किसी कारण से अंतर्निहित ब्राउज़र इवेंट की आवश्यकता है, तो इसे प्राप्त करने के लिए केवल `nativeEvent` विशेषता का उपयोग करें। प्रत्येक `SyntheticEvent` ऑब्जेक्ट में निम्न विशेषताएँ होती हैं।
```
विशेषता  -> attribute in hindi. Revisit sentence too

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,15 @@ number timeStamp
+> ध्यान दें:
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> v0.14 के अनुसार, किसी इवेंट हैंडलर से `false` वापस करना अब event propagation को रोक नहीं पाएगा। इसके बजाय, `e.stopPropagation()` या `e.preventDefault()` को ज़रूरत के अनुसार मैन्युअली स्टार्ट किया जाना चाहिए।
```
के अनुसार incorrect translation

saranshkataria replied:
v0.14 के अनुसार, किसी इवेंट हैंडलर से `false` वापस करना अब event propagation को रोक नहीं पाएगा। revisit

saranshkataria replied:
स्टार्ट incorrect translation

saranshkataria commented:
added comments till {#event-pooling}

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,15 @@ number timeStamp
+> ध्यान दें:
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> v0.14 से, किसी इवेंट हैंडलर से `false` लौटने पर event propagation अब नहीं रुकेगा। इसके बजाय, `e.stopPropagation()` या `e.preventDefault()` को ज़रूरत के अनुसार मैन्युअल रूप से चालू करना चाहिए।
```
लौटने -> lautaane

saranshkataria replied:
अब नहीं रुकेगा -> ab is not needed

saranshkataria replied:
चालू करना चाहिए। -> incorrect translation

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -6,13 +6,13 @@ layout: docs
+## ओवरव्यू {#overview}
 
-Your event handlers will be passed instances of `SyntheticEvent`, a cross-browser wrapper around the browser's native event. It has the same interface as the browser's native event, including `stopPropagation()` and `preventDefault()`, except the events work identically across all browsers.
+आपके ईवेंट हैंडलर को `SyntheticEvent`, ब्राउज़र के नेटिव ईवेंट के चारों ओर एक क्रॉस-ब्राउज़र आवरण दिया जाएगा। यह ब्राउज़र के नेटिव ईवेंट के समान है, जिसमें `stopPropagation()` और `preventDefault()` शामिल हैं, जब तक कि इवेंट्स, सभी ब्राउज़रों में समान रूप से काम करते हैं।
```
the grammar of these sentences is still a bit confusing, revisit once?

ps0305 replied:
revisited, and corrected, in case you still feels some confusion, please suggest the edit required.

saranshkataria replied:
आवरण could be wrapper in hindi instead?

saranshkataria replied:
second sentence could be reworded as: यह ब्राउज़र के नेटिव ईवेंट के समान है, जिसमें `stopPropagation()` और `preventDefault()` शामिल हैं, par yeh इवेंट्स सभी ब्राउज़रों में समान रूप से काम करते हैं

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,15 @@ number timeStamp
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` को इकट्ठा किया जाता है। इसका मतलब यह है कि `SyntheticEvent` ऑब्जेक्ट का पुन: उपयोग किया जाएगा और ईवेंट कॉलबैक लागू होने के बाद सभी प्रॉपर्टीज को nullify कर दिया जाएगा।
```
कर दिया जाएगा। -> kar diya jaata hai

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -31,15 +31,15 @@ number timeStamp
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` को इकट्ठा किया जाता है। इसका मतलब यह है कि `SyntheticEvent` ऑब्जेक्ट का पुन: उपयोग किया जाएगा और ईवेंट कॉलबैक लागू होने के बाद सभी प्रॉपर्टीज को nullify कर दिया जाएगा।
+यह परफॉरमेंस कारणों से है।
+जैसे, आप ईवेंट को asynchronous तरीके से एक्सेस नहीं कर सकते।
```
revisit the last line

saranshkataria replied:
नहीं कर सकते hain।

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -60,15 +60,15 @@ function onClick(event) {
+> ध्यान दें:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> यदि आप एक asynchronous तरीके से इवेंट के प्रॉपर्टीज का उपयोग करना चाहते हैं, तो आपको इवेंट पर `event.persist()` को कॉल करना चाहिए, जो पूल से `सिंथेटिक इवेंट` को हटा देगा और उपयोगकर्ता कोड द्वारा इवेंट के संदर्भ को बनाए रखने की अनुमति देगा।
```
`सिंथेटिक इवेंट` -> remove quotes

saranshkataria replied:
और उपयोगकर्ता कोड द्वारा इवेंट के संदर्भ को बनाए रखने की अनुमति देगा। gives the sentence a different meaning than original

saranshkataria replied:
संदर्भ  can be references in hindi since it is a technical term

saranshkataria commented:
added comments till {#supported-events}

saranshkataria commented:
@ps0305 sorry for the delay, got busy with some covid stuff. Could you please resolve the conflicts so that I can continue with the review?

ps0305 commented:
> @ps0305 sorry for the delay, got busy with some covid stuff. Could you please resolve the conflicts so that I can continue with the review?

> @saranshkataria: Conflict resolved, you can continue, and take care amid Covid situation.

saranshkataria commented:
added a few more comments

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -61,15 +61,15 @@ function onClick(event) {
+## समर्थित इवेंट्स {#supported-events}
 
-React normalizes events so that they have consistent properties across different browsers.
+React इवेंट्स को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
```
सामान्य could be left as is since the translation changes the meaning

saranshkataria replied:
normalizes is different than normal?

saranshkataria replied:
I meant for it to read normalizes, not सामान्य  and not नार्मल . Either in english/hindi works

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -61,15 +61,15 @@ function onClick(event) {
+React इवेंट्स को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
 
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+नीचे दिए गए घटना संचालकों को bubbling चरण में एक इवेंट द्वारा ट्रिगर किया गया है। कैप्चर चरण के लिए एक ईवेंट हैंडलर रजिस्टर करने के लिए, `Capture` को ईवेंट नाम में जोड़ें, उदाहरण के लिए, `onClick` का उपयोग करने के बजाय, आप कैप्चर/पकड़ना चरण में click इवेंट को संभालने के लिए `onClickCapture` का उपयोग करेंगे।
```
event handlers could be left as is in English since they are technical terms (just translate to hindi)

saranshkataria replied:
revisit paragraph as well, looks too complex

ps0305 replied:
revisited and corrected.

saranshkataria commented on content/docs/reference-events.md:
````diff
@@ -151,63 +151,60 @@ boolean shiftKey
 ```
 
-The `key` property can take any of the values documented in the [DOM Level 3 Events spec](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
+`key` प्रॉपर्टीज, [DOM स्तर 3 इवेंट्स विनिर्देश](https://www.w3.org/TR/uievents-key/#named-key-attribute-values) में प्रलेखित किसी भी वैल्यू को ले सकती है।
````
प्रॉपर्टीज should be property, and DOM Level 3 Events spec could be left as is in english

saranshkataria commented on content/docs/reference-events.md:
````diff
@@ -151,63 +151,60 @@ boolean shiftKey
 ```
 
-These focus events work on all elements in the React DOM, not just form elements.
+ये फ़ोकस इवेंट React DOM में सभी एलिमेंट्स पर काम करते हैं, न कि केवल फार्म एलिमेंट्स।
````
फार्म एलिमेंट्स par

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -151,63 +151,60 @@ boolean shiftKey
+### माउस इवेंट्स {#mouse-events}
 
-Event names:
+ईवेंट के नाम:
```
should be events

saranshkataria commented on content/docs/reference-events.md:
````diff
@@ -151,63 +151,60 @@ boolean shiftKey
 ```
 
-The `onMouseEnter` and `onMouseLeave` events propagate from the element being left to the one being entered instead of ordinary bubbling and do not have a capture phase.
+`onMouseEnter` और `onMouseLeave` ईवेंट उस एलिमेंट से प्रचारित होते हैं जिसे साधारण bubbling के बजाय किसी एक में छोड़ा जा रहा है और कैप्चर फेज नहीं है।
````
revisit grammar of sentence once

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -228,20 +225,20 @@ boolean shiftKey
+### पॉइंटर इवेंट्स {#pointer-events}
 
-Event names:
+ईवेंट के नाम:
```
same as above

saranshkataria commented on content/docs/reference-events.md:
````diff
@@ -228,20 +225,20 @@ boolean shiftKey
 ```
 
-The `onPointerEnter` and `onPointerLeave` events propagate from the element being left to the one being entered instead of ordinary bubbling and do not have a capture phase.
+`onPointerEnter` और `onPointerLeave` ईवेंट उस एलिमेंट से प्रचारित होते हैं जिसे साधारण bubbling के बजाय एक में छोड़ा जा रहा है और कैप्चर फेज नहीं है।
````
same

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -228,20 +225,20 @@ boolean shiftKey
+प्रॉपर्टीज:
 
-As defined in the [W3 spec](https://www.w3.org/TR/pointerevents/), pointer events extend [Mouse Events](#mouse-events) with the following properties:
+जैसा कि [W3 विनिर्देश](https://www.w3.org/TR/pointerevents/) में परिभाषित किया गया है, पॉइंटर इवेंट्स निम्नलिखित गुणों के साथ [माउस इवेंट्स](#mouse-events) का विस्तार करती हैं:
```
karte hain

saranshkataria commented on content/docs/reference-events.md:
````diff
@@ -256,33 +253,33 @@ string pointerType
 ```
 
-A note on cross-browser support:
+क्रॉस-ब्राउज़र समर्थन पर ध्यान दें:
````
dhyaan de -> ek note

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -256,33 +253,33 @@ string pointerType
+क्रॉस-ब्राउज़र समर्थन पर ध्यान दें:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+पॉइंटर ईवेंट्स अभी तक हर ब्राउज़र में समर्थित नहीं हैं (इस लेख को लिखने के समय, समर्थित ब्राउज़र में शामिल हैं: Chrome, Firefox, Edge और Internet Explorer)। React जानबूझकर अन्य ब्राउज़रों के लिए पॉलीफिल समर्थन नहीं करता है क्योंकि एक मानक-अनुरूप पॉलीफिल `react-dom` के बंडल आकार में काफी वृद्धि करेगा।
```
reorder to समर्थित ब्राउज़र में Chrome, Firefox, Edge और Internet Explorer शामिल हैं।

saranshkataria replied:
second sentence is too complex

saranshkataria replied:
समर्थन could be replaced by something else, rest is great :)

saranshkataria commented on content/docs/reference-events.md:
```diff
@@ -256,33 +253,33 @@ string pointerType
+पॉइंटर ईवेंट्स अभी तक हर ब्राउज़र में समर्थित नहीं हैं (इस लेख को लिखने के समय, समर्थित ब्राउज़र में शामिल हैं: Chrome, Firefox, Edge और Internet Explorer)। React जानबूझकर अन्य ब्राउज़रों के लिए पॉलीफिल समर्थन नहीं करता है क्योंकि एक मानक-अनुरूप पॉलीफिल `react-dom` के बंडल आकार में काफी वृद्धि करेगा।
 
-If your application requires pointer events, we recommend adding a third party pointer event polyfill.
+यदि आपके एप्लिकेशन को पॉइंटर इवेंट्स की आवश्यकता है, तो हम तीसरे पक्ष के पॉइंटर इवेंट्स पॉलीफिल को जोड़ने की सलाह देते हैं।
```
तीसरे पक्ष के पॉइंटर इवेंट्स पॉलीफिल को is incorrect translation

saranshkataria commented:
great work 👍 
Merging this in. Thanks!
